tests/extensions-are-available/run.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+docker-compose -f tests/extensions-are-available/docker-compose.yml down
```

Comment:
`cd` to the dir might improve readability

Reply:
ack